apps/api/docs/development-setup.md

```diff
@@ -0,0 +1,85 @@
+# Development Setup
+
+This document outlines the steps required to set up your Transcript Summarization for development. By following these steps, you'll be able to run your application locally with the necessary environment variables and database configuration.
+
+## Prerequisites
+
+Before setting up Transcript Summarization for development, ensure you have the following prerequisites with the specified versions:
+
+- **NVM (Node Version Manager):** Use NVM to manage Node.js versions.
+- **Node.js** Node.js v20.x or higher. Can be installed via `nvm` using `nvm install 20` and used with `nvm use 20`.
+- **Git:** Git v2.x or higher.
+- **MariaDB:** MariaDB v10.x or higher.
+- **Redis:** Redis v6.x or higher
+
+These prerequisites are essential for deploying and running Transcript Summarization in an environment.
+
+Please make sure to have these versions installed on your development server before proceeding with the setup.
+
+Make sure Redis and MariaDB server are up and running.
+
+```bash
+sudo systemctl status redis
+sudo systemctl status mariadb
+```
+
+## Getting Started
+
+1. Clone the repository to your local machine:
+
+   ```sh
+   git clone https://github.com/OsmosysSoftware/osm-transcript-summarizer
+   cd osm-transcript-summarizer/apps/api
+   ```
+
+2. Install project dependencies:
+
+   ```sh
+   npm install
+   ```
+
+3. Create a `.env` file in the project root and add the required environment variables:
+
+   ```env
+   # Server
+   SERVER_PORT=3000
+
+   # Node env
+   NODE_ENV=development
+
+   # Database configuration
+   DB_TYPE=mariadb
+   DB_HOST=localhost # use value as transcriptsummary-mariadb in docker
+   DB_PORT=3306
+   DB_USERNAME=root
+   DB_PASSWORD=your-password
+   DB_NAME=your-database
+   MARIADB_DOCKER_PORT=3307 # (required only if using docker)
+
+   # Redis configuration
```

Comment:
You have not added Redis Yes.

Reply:
Made changes as per suggestion, Removed the Redis related part.